simple-ipc.h

```diff
@@ -43,6 +36,13 @@ enum ipc_active_state {
 	IPC_STATE__OTHER_ERROR,
 };
 
+#ifdef SUPPORTS_SIMPLE_IPC
```

Comment:
yes, this is in my builtin-fsmonitor-part1 series currently on the mailing list.